component/identity/src/main/java/org/exoplatform/services/organization/idm/UpdateLoginTimeListener.java

```diff
@@ -71,7 +71,7 @@ public void onEvent(Event<ConversationRegistry, ConversationState> event) {
       }
 
       user.setLastLoginTime(Calendar.getInstance().getTime());
-      userHandler.saveUser(user, false);
+      userHandler.saveUser(user, true);
```

Comment:
Why is this change ? this should be a silent update with no events fired !

Reply:
To add the lastLoginTime in the profile properties, so i can filter users by connected / not connected.

Reply:
Why do you need to enable broadcasting the event ? I do not see a direct relation between the Save event on User entity and the profile update.

Reply:
yes there is a relation
https://github.com/Meeds-io/gatein-portal/blob/develop/component/identity/src/main/java/org/exoplatform/services/organization/idm/UserDAOImpl.java#L158

https://github.com/Meeds-io/social/blob/develop/component/core/src/main/java/org/exoplatform/social/core/listeners/SocialUserEventListenerImpl.java#L77

Reply:
I know that relation. I don't get it why we should relaunch all user Listeners to just save a Date property inside the user profile.
Why don't you just load the user profile and add that property in the same time as adding it to the user entity ?

Reply:
Here we are in gatein, and not haev access to userProfile, which is in social.
I think it is why @AzmiTouil use broadcast.
@AzmiTouil can you confirm ?

Reply:
Yes I confirm

Reply:
Then I believe we need to add a new listener to catch the Login event and add the property to the user profile on Social instead of broadcasting the User Update event which is too generic.

Reply:
I'm OK with @ahamdi 's proposition. Let's go to created a new listener in social